### ISSUE TITLE
Feature Request: Add weapon stats visibility for spectators

### ISSUE THREAD

**Issue by mexikoedi** [TTT-2/ttt2-weaponstats#2]
It would be very helpful especially for new players to see the weapon stats not only if they are alive but also if there are dead. 

(Maybe other stats like door, ammo,... too if it is possible.)

**Comment by mexikoedi** [comment]
Will this feature request be taken into consideration?
Otherwise this can be closed.

**Comment by TimGoll** [comment]
so you're saying specs should see targetID?

This is something I considered as well, yes

**Comment by mexikoedi** [comment]
> so you're saying specs should see targetID?
> 
> This is something I considered as well, yes

Yes that would be awesome. Especially for new players who don't know the weapons, items, ... yet.
(This was already a thing in another weapon stats addon.)
